ecosystem/README.md

```diff
@@ -0,0 +1,37 @@
+# Requirements for a successful community-centered ecosystem
+
+This section introduces the term Software Ecosystem and its relation to a community, as well as a model for quantifying its characteristics. It also lists several challenges in creating and maintaining a successful community.
+
+
+## Software Ecosystems
+
+A Software Ecosystem (SECO) is defined by van den Berk, Jansen, & Luinenburg, 2010 {{"van2010software" | cite}} to be: 'A set of actors functioning as a unit and interacting with a shared market for software and services, together with the relationships among them.' A SECO consists of a central hub, a platform and niche players. In context with the subject of this research report, the platform is the FeedbackFruits platform, the niche players are its users, mostly teachers and students, and the hub is the FeedbackFruits company. In this context, the problem description becomes: how can the hub stimulate niche creation in the SECO?
+
+Niche creation can be stimulated by determining the desired future state of the SECO {{"van2010software" | cite}}. To do this, both the current state and the desired improvements need to be known. In order for the hub to gain insight about the current state of the ecosystem, the SECO Software Assesment Model can be used. It defines a broad spectrum of characteristics of the SECO that can be made quantifiable.
+
+To gain insights about the desired improvements, some kind of knowledge management will be needed in order to aggregate the information from all of the niche players. However, due to the use of personalized learning methodologies in the FeedbackFruits platform and the broad spectrum of subjects covered by niche players, a lot of this information is tacit knowledge. In {{"mcdermott2000knowing" | cite}}, McDermott, 2000, states that sharing tacit knowledge requires interaction and proceeds to provide a list of critical factors for building a community that can share such knowledge.
```

Comment:
> In {{"mcdermott2000knowing" | cite}}, McDermott, 2000,

Zou ik van maken `In McDermott (2002)`. En dan de referentie weer naar het einde van de zin.